Python/Product/ExternalProfilerDriver/VTuneInvoker.cs

```diff
@@ -25,6 +25,27 @@
 
 namespace Microsoft.PythonTools.Profiling.ExternalProfilerDriver {
 
+    // Very simple Maybe "monad" to handle file number definitions
+    public struct Maybe<T>
```

Comment:
Seems unused? In any case, it's identical to `Nullable<T>`, which is what you seem to be using already.